controls.py

```diff
@@ -15,23 +13,21 @@ def Number(id, default, min, max):
     ----------
     id : str
         ID used for Dash callbacks.
-    default : int
+    default : float
         Default value that is displayed in the input box when user loads the page.
-    min : int
-        Minimum value the user can select from.
-    max : int
-        Maximum value the user can select from.
+    min : float, optional
+        Minimum value the user can select from. Default is None.
+    max : float, optional
+        Maximum value the user can select from. Defautl is None.
+
     Returns
     -------
     dbc.Input : component
         dbc Input component with numerical inputs.
-    Notes
-    -----
-    Examples
-    --------
     """
-    return dbc.Input(id=id, type="number", min=min, max=max, size="sm",
-                     className="m-1 d-inline-block w-auto",debounce=True,value=str(default))
+    return [dbc.Input(id=id, type="number", min=min, max=max, size="sm",
```

Comment:
Do we think that always returning a list of components is the way to go? (This is an actual, no rhetorical question: I've gone back and forth)

Reply:
Number doesn't work without the []

Reply:
Well right, because the places that use it are expecting lists. I'm just wondering more broadly whether this is the right way to do things. It's probably fine for the sake of this PR though